server/server.go

```diff
@@ -100,13 +101,20 @@ func (s *Server) Start(ctx context.Context) error {
 
 	logger := logging.FromContext(ctx)
 
+	// grpc metrics
+	metrics := grpc_prometheus.NewServerMetrics(
+		grpc_prometheus.WithServerHandlingTimeHistogram(
+			grpc_prometheus.WithHistogramBuckets(prometheus.ExponentialBuckets(0.0001, 2, 20)),
```

Comment:
100us is too low probably, maybe 10ms too

Reply:
I picked a low value as this is also reported by other endpoints, which are faster. 

Changed to 1ms as well.